src/study_plans/services.py

```
@@ -273,6 +244,97 @@ def get_study_plan(self, plan_id: str, workspace_info: Dict = None) -> Optional[
+            modules = list(get_db().modules.find({"study_plan_id": ObjectId(plan_id)}))
+            module_ids = [module["_id"] for module in modules]
+            
+            # Inicializar contadores para el resumen
+            total_modules_deleted = 0
+            total_topics_deleted = 0
+            total_evaluations_deleted = 0
+            
+            # 1. Eliminar todos los temas asociados a los módulos
+            if module_ids:
+                topics_result = get_db().topics.delete_many({"module_id": {"$in": module_ids}})
+                total_topics_deleted = topics_result.deleted_count
```

Ruff (0.12.2)

Local variable `total_topics_deleted` is assigned to but never used

Remove assignment to unused variable `total_topics_deleted`

(F841)